### PR TITLE
syncer: remove improper ignored error

### PR DESCRIPTION
**Pull request by lance6716**
### What problem does this PR solve? 
close https://github.com/pingcap/dm/issues/1392

### What is changed and how it works?
3 errors should not be ignored: ErrDatabaseNotExists, ErrTableNotExists, ErrColumnNotExists. These errors are not caused by re-execute DDL.

### Check List 

Tests 

 - Integration test

Related changes

 - Need to cherry-pick to the release branch
 - Need to be included in the release note

### CODE REVIEW

File: tests/all_mode/run.sh [hunk 1]
```diff
@@ -172,11 +172,11 @@ function test_stop_task_before_checkpoint(){
 function run() {
     run_sql_both_source "SET @@GLOBAL.SQL_MODE='ANSI_QUOTES,NO_AUTO_VALUE_ON_ZERO'"
 
-    test_session_config
-
-    test_query_timeout
-
-    test_stop_task_before_checkpoint
+#    test_session_config
```
Comment (GMHDBJD) [inline tests/all_mode/run.sh:175]: remove debug comment

**Comment by GMHDBJD** [comment]
Will re-execute `DROP COLUMN/TABLE/DATABASE` cause the not exist error?

**Comment by lance6716** [comment]
> Will re-execute `DROP COLUMN/TABLE/DATABASE` cause the not exist error?

I think that should cause ErrDatabaseDropExists ErrTableDropExists ErrCantDropFieldOrKey

**Pull request merged**